tests/crypto/test_cases/CMakeLists.txt

```diff
@@ -41,13 +42,16 @@ zephyr_sources_ifdef(CONFIG_MBEDTLS_ECDSA_C         test_vectors_ecdsa_random.c)
 zephyr_sources_ifdef(CONFIG_MBEDTLS_ECDSA_C         test_vectors_ecdsa_sign.c)
 zephyr_sources_ifdef(CONFIG_MBEDTLS_ECDSA_C         test_vectors_ecdsa_verify.c)
 zephyr_sources_ifdef(CONFIG_MBEDTLS_ECJPAKE_C       test_vectors_ecjpake.c)
+if(NOT CONFIG_CRYPTO_SKIP_HASH_TESTS)
 zephyr_sources_ifdef(CONFIG_MBEDTLS_SHA256_C        test_vectors_hkdf.c)
 zephyr_sources_ifdef(CONFIG_MBEDTLS_SHA256_C        test_vectors_hmac.c)
 zephyr_sources_ifdef(CONFIG_MBEDTLS_SHA256_C        test_vectors_sha_256.c)
 zephyr_sources_ifdef(CONFIG_MBEDTLS_SHA512_C        test_vectors_sha_512.c)
+endif()
 
 if(CONFIG_MBEDTLS_CMAC_C OR CONFIG_MBEDTLS_CMAC_ALT)
```

Comment:
Please indent the lines between this `if()` and the following `endif()`

Reply:
Indented